src/routes/Intro/Intro.js

```diff
@@ -278,6 +282,12 @@ const Intro = ({ queryParams }) => {
             core.transport.off('CoreEvent', onCoreEvent);
         };
     }, [routeFocused]);
+
+    if(profile?.auth?.key) {
```

Comment:
Could you wrap this in useEffect with [] as deps so it gets executed only on mount?

Reply:
wrapping it to an `useEffect` the login page will render until the `profile` exists, take a look at it:

https://user-images.githubusercontent.com/51221635/222966364-8fd2dcb5-8f2f-484a-9ba8-22ed093beece.mp4

Reply:
> Could you wrap this in useEffect with [] as deps so it gets executed only on mount?

hello @nklhtv I think that the `useEffect` is not needed in the conditional render. 

please check the documentation https://beta.reactjs.org/learn/conditional-rendering and give me your thoughts

Reply:
I will take a look and back to you ASAP

Reply:
Hey @nklhtv thanks for your suggestions. Infortunatelly it is not too much clear to me, I just update this PR with a WIP pseudo-code implementation as you suggest and I'm not sure how to check if the user is authenticated or not properly

Reply:
@vczb  you can use `profile.auth` i think this is what @nklhtv  meant like here auth is null so display stuff for anonymous user
https://github.com/Stremio/stremio-web/blob/d44f7047dcee5f0b2257318ac7153a09bbfe2959/src/common/NavBar/HorizontalNavBar/NavMenu/NavMenuContent.js#L44